docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -38,112 +28,136 @@ If you install from the APT repository, then Grafana is automatically updated ev
+   sudo apt-get install grafana-enterprise
+   ```
 
-Download the latest [`.tar.gz` file](https://grafana.com/grafana/download?platform=linux) and extract it. The files extract into a folder named after the Grafana version downloaded. This folder contains all files required to run Grafana. There are no init scripts or install scripts in this package.
+## Install Grafana using DEB or the standalone binaries
 
-```bash
-wget <tar.gz package url>
-sudo tar -zxvf <tar.gz package>
-```
+If you choose not to install Grafana using APT, you can download and install Grafana using DEB or the standalone binaries.
+
+### Before you begin
+
+To follow these instructions, you need:
+
+- xxx
+- xxx
+
+### Steps
+
+Complete the following steps to install Grafana using DEB or the standalone binaries:
+
+1. Navigate to the [Grafana download page](https://grafana.com/grafana/download).
+1. Select the Grafana version you want to install.
+   - The most recent Grafana version is selected by default.
+   - The **Version** field displays only tagged releases. If you want to install a nightly build, click **Nightly Builds** and then select a version.
+1. Select an **Edition**.
+   - **Enterprise:** This is the recommended version. It is functionally identical to the open source version, but includes features you can unlock with a license, if you so choose.
+   - **Open Source:** This version is functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
+1. Depending on which system you are running, click the **Linux** or **ARM** tab on the download page.
+1. Copy and paste the code from the installation page into your command line and run.
 
 ## 2. Start the server
 
-This starts the `grafana-server` process as the `grafana` user, which was created during the package installation.
+The following sections provide instructions for starting the `grafana-server` process as the `grafana` user, which was created during the package installation.
 
 If you installed with the APT repository or `.deb` package, then you can start the server using `systemd` or `init.d`. If you installed a binary `.tar.gz` file, then you need to execute the binary.
 
-### Start the server with systemd
+### Start the Grafana server with systemd
 
-To start the service and verify that the service has started:
+Complete the following steps to start the Grafana server with systemd and verify that it is running:
 
-```bash
-sudo systemctl daemon-reload
-sudo systemctl start grafana-server
-sudo systemctl status grafana-server
-```
+1. To start the service, run the following commands:
 
-Configure the Grafana server to start at boot:
+   ```bash
+   sudo systemctl daemon-reload
+   sudo systemctl start grafana-server
+   sudo systemctl status grafana-server
+   ```
 
-```bash
-sudo systemctl enable grafana-server.service
-```
+1. To verify that the service is running, run the following command:
```

Comment:
@DanCech , the original content makes mention of verifying that the server is up and running, which I think is a good idea. But, there were no instructions for how to do that. Would you know what to put here?

Reply:
`sudo systemctl status grafana-server` is the command that will tell the user whether the service is running